parsers/PH.py

```diff
@@ -611,15 +616,11 @@ def aggregate_per_datetime_mode(df: pd.DataFrame) -> pd.DataFrame:
     return df.groupby(["datetime", "mode"]).sum(numeric_only=True).reset_index()
 
 
-def pivot_per_mode(df: pd.DataFrame) -> pd.DataFrame:
+def add_production_mix_column(df: pd.DataFrame) -> pd.DataFrame:
     df = df.copy()
-    df = df.pivot(index="datetime", columns="mode")
-    # Flatten columns and make them "production.{mode}"
-    df.columns = df.columns.to_series().str.join(".")
-    # Handle storage
-    if "production.hydro_storage" in df.columns:
-        df = df.rename(columns={"production.hydro_storage": "storage.hydro"})
-        df["storage.hydro"] *= -1
+    df["production_mix"] = None
```

Comment:
I don't think the name of the column is self-descriptive.

If I understand correctly, we end up with:

```
idx, mode, value, production_mix 
dt, coal, 1239, "production"
```

If that's the case, then I would second Viktor's comment: do we even need that, and can't we directly use the mode column for that info?